systemd/system/sysbox.service

```diff
@@ -0,0 +1,26 @@
+# Sysbox unit to act as a wrapper of sysbox's inner components/daemons:
+# sysbox-mgr and sysbox-fs.
+
+[Unit]
+Description=Sysbox container runtime
+Documentation=https://github.com/nestybox/sysbox
+# rahul: using BindsTo + PartOf prevents the services that are bound
+# from restarting on failure. For example, if sysbox-mgr.service is
+# killed, sysbox.service will stop because of the 'BindsTo' constraint.
+# However, because sysbox-mgr.service is 'PartOf' sysbox.service,
+# sysbox-mgr.service will not restart until sysbox.service has started
+# BindsTo=sysbox-mgr.service sysbox-fs.service
```

Comment:
For reviewers: note this change to service file, restarts don't work without removing the BindsTo constraint
The effect of this is that if sysbox-mgr or sysbox-fs stop/restart, sysbox.service won't. I don't think this is an issue but just flagging